<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose OS (Exclusive)</string>
   </property>
   <widget class="QCheckBox" name="winkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mackBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="linuxBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>RadioButton</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose uC (NonExclusive)</string>
   </property>
   <widget class="QCheckBox" name="winkBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>STM32</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mackBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AVR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="linuxBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Grab Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>411</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Set State</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
